alws/utils/multilib.py

```diff
@@ -249,11 +249,13 @@ async def update_module_index(module_index: IndexWrapper, module_name: str,
                                   module_stream: str, packages: list):
         if not packages:
             return
-        if module_index.has_devel_module() and 'python' not in module_name:
+
+        # Ruby and Python modules should have multilib in their main module
+        if 'ruby' in module_name or 'python' in module_name:
```

Comment:
I'm think that main problem here, because we have in index usual and devel model, can we process modules successively?

Reply:
Yes, before having devel modules for only a handful of modules logic was correct. But now we have devel modules for all and multilib for ruby started acting out. I've put a specific condition for ruby to move multilib into the main module